src/sst/core/model/element_python.h

```diff
@@ -171,7 +171,12 @@ class SSTElementPythonModule {
     /// \param filename filname used when reporting errors
     ///
     ///     
+#ifdef SST_CONFIG_HAVE_PYTHON3
```

Comment:
Let's just require the filename for both.  I think the only instance of this being used in elements includes a filename (and debugging without the filename can be difficult).